digitalocean/app/app_spec.go

```diff
@@ -1,12 +1,11 @@
 package app
 
 import (
-	"log"
-	"net/http"
-
 	"github.com/digitalocean/godo"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"log"
+	"net/http"
```

Comment:
```suggestion
	"log"
	"net/http"

	"github.com/digitalocean/godo"
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
```

^^ should fix lint ci issue!